### PR TITLE
[IMP]10702 add module mrp_production_workcenter_line_reporting

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by Rona111** [comment]
cc @elicoidal @siyuancn @seb-elico @duanyp1991

File: mrp_production_workcenter_line_reporting/README.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================================
+Mrp productoin workcenter line reporting
```
Comment (elicoidal) [inline mrp_production_workcenter_line_reporting/README.rst]: MRP
Production

File: mrp_production_workcenter_line_reporting/README.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+========================================
+
+This model allows to record the finished and scraped quantity
```
Comment (elicoidal) [inline mrp_production_workcenter_line_reporting/README.rst]: summary is not consistent with title

File: mrp_production_workcenter_line_reporting/README.rst [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+To install this module, you need to:
+
+ * have basic modules installed (mrp_operations)
+
+
+Usage
+=====
+
+A new button is added in the page "work order" under "Manufacturing Orders".
```
Comment (elicoidal) [inline mrp_production_workcenter_line_reporting/README.rst]: I am confused

**Comment by noahzaozao** [comment]
@elicoidal another module is depend on this module mrp_production_workcenter_line_reporting
https://github.com/Elico-Corp/odoo-addons/pull/42
shall we open this pr again ?

**Comment by elicoidal** [comment]
ok @noahzaozao

**Comment by noahzaozao** [comment]
@elicoidal please review this pr again

**Pull request merged**